tools/badctl/cmd/gen_test.go

```diff
@@ -0,0 +1,41 @@
+package cmd
```

Comment:
bad test file, because we want to have readable scenario like:

generateDockerFiles should generate all wanted file
copyDir should stop if os.IsNotExists
(...)

I don't see any test case in your test, for me you don't check success or errors cases.